docs/production-deployment/cloud/nexus/using-nexus.mdx

```diff
@@ -0,0 +1,99 @@
+---
+id: using-nexus
+slug: /cloud/nexus/using-nexus
+title: Using Temporal Nexus
+description: Discover how to implement and use Nexus within Temporal workflows, including creating Nexus Services, Endpoints, and cross-team collaboration.
+sidebar_label: Using Temporal Nexus
+tags:
+  - temporal cloud
+  - nexus
+
+keywords:
+  - cross-team collaboration
+  - cross-Namespace workflows
+---
+
+Nexus Services are exposed from a Nexus Endpoint that is created in the Nexus API Registry.
+Nexus Endpoints include service documentation that enables others to use the Nexus Services provided by a Nexus Endpoint, for example from caller Workflows.
+
+![NexusAPIRegistry](/img/NexusAPIRegistry.png)
+
+Temporal has built-in Nexus Machinery that performs the low-level [Nexus RPC](/glossary#nexus-rpc) operations on the wire and provides an integrated Temporal SDK experience to create Nexus Services in a handler Worker and use them from a caller Workflow through a Nexus Endpoint.
+
+In Temporal Cloud, the Nexus Machinery routes Nexus requests across Namespace boundaries through a secure mTLS Envoy mesh, where they are sync matched to a Worker in the handler Namespace, which processes the Nexus request.
+
+Nexus caller Workflows will send commands to Temporal Cloud to schedule or cancel a Nexus Operation in the caller Namespace.
+Handler Workers will poll for Nexus tasks in the handler Namespace.
+
+Nexus Operation handlers can be implemented as [Asynchronous Operations](/nexus/services-operations#asynchronous-operation-lifecycle) or [Synchronous Operations](/nexus/services-operations#asynchronous-operation-lifecycle), using Temporal SDK builders that enable running a Nexus Service in a Worker, typically the same Worker as the underlying primitives the Nexus Service abstracts.
+
+For example, an asynchronous operation that starts a Workflow in a different Namespace:
+
+Unlike a traditional RPC, an arbitrary-duration Nexus Operation has an identity (ID) that lasts beyond the process lifetimes of the caller and handler, and can be used to re-attach to a long-lived Nexus Operation, for example one backed by a Temporal Workflow.
```

Comment:
this was a bit hard to grok. What is this trying to communicate? I don't think we have the capability to attach a new Workflow to wait on an existing nexus async operation, right?